src/docker-object-model.ts

```diff
@@ -0,0 +1,33 @@
+import { ToolRunner, IExecOptions, IExecSyncResult } from "./utilities/tool-runner";
+
+export class DockerExec{

```

Comment:
Recommend running some formatting tool for all the files.

Reply:
Done